README.md

```diff
@@ -19,6 +18,26 @@ But when you need to scale your application Caddy is limited to its static confi
 To overcome this issue we are using `docker-gen` to generate configuration everytime a container spawns or dies.
 Now scaling is easy!
 
+## CADDY 2
+
+BREAKING CHANGES HERE!
```

Comment:
Let's add `Breaking change since <v...>` with the link to the previous version.

Reply:
:heavy_check_mark: